### PR TITLE
Implement Relay

### PR DESCRIPTION
**Pull request by shanecurran**

### CODE REVIEW

**Review by lfarrel6 — CHANGES_REQUESTED** [review]
Needs version bump

File: .gitignore [hunk 1]
```diff
@@ -112,4 +112,6 @@ dist
 .debug
 
 # MAC
-*.DS_Store
\ No newline at end of file
+*.DS_Store
```
Comment (eoinboylan) [inline .gitignore:115]: This should be in your global .gitignore on your laptop.

File: lib/config.js [hunk 2]
```diff
@@ -1,26 +1,27 @@
-const { version } = require('../package.json');
```
Comment (eoinboylan) [inline lib/config.js:1]: Half the colons are single half double. Dont we use single for everything except sting interpolation?

**Review by eoinboylan — DISMISSED** [review]

File: lib/config.js (hunk 2 above)
Comment (shanecurran) [reply lib/config.js:1]: Yeah, sorry — think my linter is misconfigured. Will fix

**Review by eoinboylan — APPROVED** [review]

**Review by lfarrel6 — APPROVED** [review]

**Pull request merged**